flask_testing/test_config.py

```diff
@@ -0,0 +1,109 @@
+import unittest
+import os
+from app.config import app,db
+
+from app.models.allnewsmodel import AllNewsModel
+
+from os.path import abspath, dirname, join
+
+project_dir = os.path.dirname(os.path.abspath(__file__))
+
+'''SECRET_KEY = 'flask-session-insecure-secret-key'
+SQLALCHEMY_DATABASE_URI = 'sqlite:///' + join(_cwd, 'flask-tracking.db')
+SQLALCHEMY_ECHO = True'''
+
+
+class BaseConfiguration(object):
+    DEBUG =False
+    TESTING = False
+    SECRET_KEY = 'flask-session-insecure-secret-key'
+
+class TestConfiguration(BaseConfiguration):
+    TESTING = True
+    WTF_CSRF_ENABLED = False
+
+    database_file = "sqlite:///{}".format(os.path.join(project_dir,"db/newsdatabase.sqlite"))
+    SQLALCHEMY_DATABASE_URI = database_file
+    # Since we want our unit tests to run quickly
+    # we turn this down - the hashing is still done
+    # but the time-consuming part is left out.
+    HASH_ROUNDS = 1
+
+from flask import url_for
+
+from test_base import BaseTestCase
+from models.allnewsmodel import AllNewsModel
+
+
+class UserViewsTests(BaseTestCase):
+    def test_allnews(self):
+        AllNewsModel.create(
```

Comment:
For test data I like the idea of having "close to reality" data, but it's also a good idea to not have very long data. It just fluffs up your test and increases the chance of mistaking something during copy-paste (hence making your tests more fragile)